config/application.rb

```diff
@@ -49,5 +49,9 @@ class Application < Rails::Application
       base_controller_class: "SupportInterface::SupportInterfaceController"
     }
     config.console1984 = { ask_for_username_if_empty: true }
+
+    [ENV.fetch("HOSTING_DOMAIN"), ENV.fetch("CHECK_RECORDS_DOMAIN")].each do |domain|
+      config.hosts << domain
+    end
```

Comment:
Instead of this, we should probably just have the following entries in config/environments/dev, to complement the ones already in config/environments/test:

```
 config.hosts << "check_records.localhost"
 config.hosts << "qualifications.localhost"
```

Reply:
> What if the ENV variables don't match?

Could you potentially do this instead:
```
  config.hosts << ENV.fetch("CHECK_RECORDS_DOMAIN")
  config.hosts << ENV.fetch("HOSTING_DOMAIN")
```
That appears to boot just fine with the correct values being set.

> How would you handle setting these in production?

It seems that this is empty in prod [by default](https://guides.rubyonrails.org/configuring.html#actiondispatch-hostauthorization), and setting it introduces a `Host` header check. If we opt to start doing that I think we should consider it in a separate PR, but it doesn't look like we have a motivation to do so currently.